tests/unit/tasks/test_generate_queries.py

```diff
@@ -690,7 +738,9 @@ def partition_stub(values, columns, table_name="test_table"):
     }
 
 
-def table_stub(columns, partition_keys, table_name="test_table"):
+def table_stub(
+    columns, partition_keys, table_name="test_table", partition_key_types="string"
```

Comment:
nit: types? shouldn't this be type if the value isn't a list?

Reply:
you're right, renamed to partition_keys_type